assemblies/core/lib/pom.xml

```diff
@@ -502,12 +500,11 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>${slf4j-api.version}</version>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-log4j12</artifactId>
-            <version>${slf4j-log4j12.version}</version>
+            <version>${slf4j.version}</version>
```

Comment:
The version is being managed at the parent pom level. No need to define a version here.